test/jdk/java/math/BigDecimal/ValueOfDouble.java

```diff
@@ -0,0 +1,47 @@
+import org.junit.jupiter.api.Test;
+
+import java.math.BigDecimal;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+/**
+ * @test
+ * @run junit ValueOfDouble
+ */
+
+public class ValueOfDouble {
+    private static final String DIGITS = "1234567899123456789"; // Enough digits to fill a long
```

Comment:
Should we consider including 0 in this list?

This file needs a license header, and the jtreg directive should be `/*` instead of `/**` and usually placed before imports, and usually we include a `@bug` tag (once there is an issue ID) and a `@summary` tag.

Reply:
I left out 0 specifically to ensure the all  lengths  were tried. The loop over the exponents effectively adds zeros afterwards.